include/hw_unique_key.rst

```diff
@@ -0,0 +1,81 @@
+.. _lib_hw_unique_key:
+
+Hardware unique key
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The hardware unique key library manages symmetric root keys on devices with the Arm CryptoCell security subsystem.
+
+Functionality
+*************
+
+Hardware unique keys (HUKs) are device-specific keys that can be used with functions for key derivation to generate other keys.
+The generated keys can then be used by applications for symmetric encryption or authentication.
+By using HUKs, you can let your application use multiple keys without having to store them on the device, as they can be recreated from a HUK when needed.
```

Comment:
Maybe change "recreated" to "derived" to put more emphasis on the key derivation function

Note: You do have to "store" the label information to be able to recreate the keys. generally the label information can be kept in code, so storing is not necessarily the right term

I don't know how much we need to explain this in the current context so I leave it up to you what is necessary to state here

Reply:
```suggestion
By using HUKs, you can let your application use multiple keys without having to store them on the device, as they can be recreated from a HUK, using a static label, when needed.
```

Reply:
fixed